File: 1446-angle-between-hands-of-a-clock/angle-between-hands-of-a-clock.cs
public class Solution {
    public double AngleClock(int hour, int minutes) {
        double smallAngle = 0;
        double hourAngle = 0;
        double otherAngle = 0;
        if(hour == 12 && minutes == 00)
        {
            hourAngle = 0;
        }
        else{
            hourAngle = hour * 30 + minutes * 0.5;
        }

        double minuteAngle = minutes * 6;

        smallAngle = Math.Abs(hourAngle - minuteAngle);
        otherAngle = 360 - smallAngle;
        return smallAngle < otherAngle ? smallAngle : otherAngle;
    }
}

File: 1446-angle-between-hands-of-a-clock/angle-between-hands-of-a-clock.cpp
class Solution {
public:
    double angleClock(int hour, int minutes) {
        if (hour == 12) hour = 0;
        if (minutes == 60) minutes = 0;

        double hourAngle = (hour * 30) + (0.5 * minutes);
        double minuteAngle = 6 * minutes;

        double angle = abs(minuteAngle - hourAngle);

        return min(angle, 360 - angle);
    }
};